end-user-application-machines/new-agent-vm/rhel-standard-agent.json

```diff
@@ -246,7 +246,7 @@
                 "typeHandlerVersion": "2.0",
                 "autoUpgradeMinorVersion": true,
                 "settings": {
-                    "fileUris": [ "[concat(parameters('_artifactsLocation'), '/', variables('scriptFileName'))]" ]
+                    "fileUris": [ "[concat(parameters('_artifactsLocation'), '/', variables('scriptFileName'), parameters('_artifactsLocationSasToken'))]" ]
```

Comment:
That's interesting - I wonder if this works under 'protectedsettings' as well now that we have the token.

Reply:
i got this from examples online but good point, i'll try it from protected settings